chaco/base_contour_plot.py

```diff
@@ -153,6 +153,7 @@ def _value_data_changed_fired(self):
     def _index_mapper_changed_fired(self):
         # If the index mapper has changed, then we need to redraw
         self.invalidate_draw()
+        self.request_redraw()
```

Comment:
Seems to work. Not sure what `self.invalidate_draw()` does as the zoom works even without it.